cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example

```diff
@@ -65,13 +65,15 @@ instances:
   #
   # wallet: <WALLET_DIR>>
 
-  ## @param instant_client - boolean - optional
-  ## Force using instant_client even when 'tns_alias' isn't used. This might be necessary
+  ## @param oracle_client - boolean - optional
```

Comment:
I would suggested to keep `@param instant_client` and mark it as `DEPRECATED, use oracle_client instead`, then add `oracle_client` as a new config option.

Reply:
Done.